cms/static/js/spec/views/pages/container_spec.js

```diff
@@ -337,28 +337,37 @@ define(["jquery", "underscore", "underscore.string", "common/js/spec_helpers/aja
                             EditHelpers.confirmPrompt(promptSpy, clickNo);
                         };
 
-                        deleteComponent = function (componentIndex, requestOffset) {
+                        deleteComponent = function (componentIndex, options) {
```

Comment:
Good catch. I had options for a while, but then refactored them into the ``globalPageOptions`` because they needed to vary per suite configuration. I've pushed up a fix to remove ``options`` in both methods.